app/controllers/announcements_controller.rb

```diff
@@ -53,7 +53,7 @@ def scheduled_announcements
 
   def redirect_to_news_and_communications(format = "")
     base_path = "#{Plek.new.website_root}/search/news-and-communications#{format}"
-    redirect_to("#{base_path}?#{news_and_communications_query_string}")
+    redirect_to("#{base_path}?#{news_and_communications_query_string}", allow_other_host: true)
```

Comment:
Not something to fix now, but I wonder why these URLs have a host name in them when they could just be a redirect to a path. It's somewhat confusing how base_path is a full URL rather than a path 🤔

Reply:
It feels like maybe `website_root` ought to return `/` when it's being asked for the same instance rather than a different environment. For that matter I wonder if we even need to invoke Plek at all for this, unless there are cases I'm overlooking.

Reply:
Yeah I think we just don't need to involve Plek here. I can see it's done a few times though so I'm wondering if there was a reason why someone did it - commit message doesn't seem to explain why :-/

Reply:
I can only think that if `GOVUK_WEBSITE_ROOT` was ever anything but a hostname the distinction would matter (i.e., `http://www.dev.gov.uk/some-prefix`. But that doesn't seem to be the case anywhere I can find.